src/ast/transform/IOAttributes.h

```diff
@@ -89,10 +87,9 @@ class IOAttributesTransformer : public Transformer {
             }
 
             // Casting due to json11.h type requirements.
-            long long arity{static_cast<long long>(rel->getArity() - auxArityAnalysis->getArity(rel))};
-            long long auxArity{static_cast<long long>(auxArityAnalysis->getArity(rel))};
+            long long arity{static_cast<long long>(rel->getArity())};
```

Comment:
DRY?  `auto` perhaps?

Reply:
I'm keeping it consistent with the rest of the code in the file for now - I just removed the auxiliary arity field. A future PR might look into fixing up this code quality issue in the file.